src/Components/Shared/ConfirmModal/modalConfirm.module.css

```
@@ -1,6 +1,6 @@
 .modal {
   position: fixed;
-  z-index: 1;
+  z-index: 999;
```

we should not modify the styles of the shared components

```suggestion
  z-index: 1;
```